src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Categories/FormTheme/category.html.twig

```diff
@@ -0,0 +1,62 @@
+{% extends '@PrestaShop/Admin/TwigTemplateForm/prestashop_ui_kit.html.twig' %}
+
+{% block category_seo_preview_widget %}
+    <div id="serp-app" data-category-url="{{ data }}"></div>
+    <small class="form-text">
+      {{ 'Here is a preview of how your page will appear in search engine results.'|trans({}, 'Admin.Global') }}
+    </small>
+{% endblock category_seo_preview_widget %}
+
+{% block image_with_preview_widget %}
```

Comment:
Since `ImageWithPreviewType` seems to be a form type generic (it's not in the category namespace) even though for now it's only used in category, I think this block should be moved into the generic form theme `prestashop_ui_kit_base.html.twig` file (only the `image_with_preview_widget` block)

Reply:
Sure I guess it makes sense. But I didn't build it with any other use cases in mind other then category page specifically needs. So if somebody uses it outside of it, it can be a bit tricky.